pilot-app/src/components/form/fields/wimTreeselect.vue

```diff
@@ -0,0 +1,68 @@
+﻿<template>
+    <div :class="fieldClass(field)" >
+        <label v-if="prefix(field)" v-html="prefix(field)"></label>
+        <treeselect v-model="field.value"
```

Comment:
Codacy has a fix for the issue: ['v-model' should be on a new line.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
        <treeselec
```